src/components/ButtonPanel.js

```diff
@@ -3,33 +3,33 @@ import Button from './Button';
 
 const ButtonPanel = () => (
   <div>
```

Comment:
- `ButtonPanel` component should use flex display mode as mentioned on Pathwright, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png)